cli/lsp/language_server.rs

```diff
@@ -377,7 +377,7 @@ impl LanguageServer {
       let mut ls = self.0.write().await;
       if let Ok(configs) = configs_result {
         for (value, internal_uri) in
-          configs.into_iter().zip(specifiers.into_iter().map(|s| s.1))
+          configs.into_iter().zip(specifiers.into_iter().map(|s| s.0))
```

Comment:
Maybe `specifiers` should be a struct with descriptive field names instead of a tuple?

Reply:
Yes, I opened https://github.com/denoland/deno/issues/18374 for the future